### ISSUE TITLE
Sequence 

### ISSUE THREAD

**Issue by khankuan** [chenglou/react-motion#236]
Hi, I have a list view and a desired animation in this sequence:
1) item is delete, it fades out to 0.
2) items are rearranged, where items below the deleted item translates upwards.

How can I achieve a "sequence" of animations?

Thanks!

**Comment by nkbt** [comment]
It looks like a duplicate of #224 and blocked by #235

**Comment by chenglou** [comment]
Yep. Closing in favour of #224!
